Xplat/src/main/java/vazkii/patchouli/client/book/template/VariableAssigner.java

```diff
@@ -169,7 +171,7 @@ private static IVariable inv(IVariable arg) {
 	}
 
 	private static IVariable stacks(IVariable arg) {
-		return IVariable.from(arg.as(Ingredient.class).getItems());
+		return IVariable.from(arg.as(Ingredient.class).getItems(), RegistryAccess.fromRegistryOfRegistries(BuiltInRegistries.REGISTRY));
```

Comment:
is this ok? ingredients can use components and those can reference dynamic registries.